README.md

```diff
@@ -48,4 +48,29 @@ IOs are lazy so they don't actually do anything until they are interpreted.
 `promisedResult` will return a promise of a FiberResult.
 
 ## Concurrency Abstractions
-Waveguide also provides Ref (synchronous mutable cell), Deferred (set once asynchronous cell), Semaphore, and an asynchronous Queue implementation.
+Waveguide also provides Ref (synchronous mutable cell), Deferred (set once asynchronous cell), Semaphore, and an asynchronous queue implementation.
+
+## Compared to...
+
+### funfix
+[funfix](https://github.com/funfix/funfix/) provides a whole suite of functional tools in its core such as either, option, and others.
+In contrast, waveguide has a peer dependency on [fp-ts](https://github.com/gcanti/fp-ts/) for this machinery and exports instances for fp-ts typeclasses where appropriate.
+
+Waveguide does not attempt to provide an equivalent to funfix's weaker primitives such as Eval or Future. 
+Waveguide does not provide an equivalent to funfix's Scheduler abstraction at this time.
+Both waveguide and funfix provide continuous synchronous execution up until an asynchronous boundary. 
```

Comment:
By that I think you're referring to the execution model which continues to execute chains of map / flatmap optimistically.
I known funfix batches the calls (to provide fairness) does waveguide does the same?

Reply:
waveguide doesn't provide any fairness guarantees. Its like to the cats-effect model where you have to insert an IO.shift (here called yield_) to get an async boundary and achieve fairness. I can elaborate on this more, but I was specifically thinking about cases like requestAnimationFrame where doing auto-batching would be a giant foot gun if someone wants to perform a bunch of synchronous effects against the DOM.

Reply:
@rzeigler Ok so maybe we should state it does not provides fairness for that very reason.